lib/src/functions/color.dart

```diff
@@ -47,16 +47,20 @@ final global = UnmodifiableListView([
     }
   }),
 
-  BuiltInCallable("invert", r"$color, $weight: 50%", (arguments) {
+  BuiltInCallable("invert", r"$color, $weight: 100%", (arguments) {
```

Comment:
No, because previously this had a special case for `$weight: 50%` where it treated it as though the weight was `100%` :stuck_out_tongue_closed_eyes:. We removed that special case, but actually calling `invert()` with one argument still works the same as it always did.